specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/Clusters-preview.json

```diff
@@ -37,7 +37,7 @@
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/clusters": {
       "get": {
         "tags": [
-          "Event Hubs Clusters"
+          "Clusters"
```

Comment:
I will suggest to keep "Event Hubs Clusters" as it is, "Cluster" means something different in Azure terminology

Reply:
@mythri1006 can you please make this change ?

Reply:
Reverted the tag name changes as informed above